tests/proxy_models/models.py

```diff
@@ -4,6 +4,7 @@
 than using a new table of their own. This allows them to act as simple proxies,
 providing a modified interface to the data from the base class.
 """
+from django.contrib.auth.models import User as AuthUser
```

Comment:
Why do we need the alias here?

Reply:
There are already a `class User` and `class UserProxy` in the models...